src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -241,10 +242,12 @@ public static String getDocumentRequest(CaseData caseData, String accessKey)  th
      * @param caseData data for the case
      */
     public static void createDynamicListSelection(CaseData caseData) {
-        DynamicFixedListType dynamicRespondent = DynamicFixedListType.from(
-                DynamicListHelper.createDynamicRespondentName(caseData.getRespondentCollection()));
+        DynamicFixedListType dynamicList = DynamicFixedListType.from(DynamicListHelper.createDynamicRespondentName(
+                caseData.getRespondentCollection().stream()
```

Comment:
do we need to check if respondentCollection is null?

Reply:
@GauravChawlaMOJ null check added and error message. Created a unit test to reflect as well